src/utils/sortObjectKeys.ts

```diff
@@ -10,11 +13,16 @@ const sortHandlers = {
     Object.fromEntries(
       Object.entries(object).sort(([a], [b]) => b.localeCompare(a))
     ),
+  [SortOrder.NONE]: (object: GenericObject) => object,
```

Comment:
maybe this is not needed, I mean it is not really used, it's only here to avoid it breaks in line 22

I think it is better cover the case on the function sortObjectKeys

```suggestion
```

Reply:
Added more info in this comment: https://github.com/heliomar-pena/i18n-populator/pull/88/files#r2024811205